src/dialects/abstract/query-interface.js

```diff
@@ -276,7 +276,7 @@ export class QueryInterface {
    * @returns {Promise}
    */
   async dropTable(tableName, options = {}) {
-    options.cascade = (options.cascade && this.queryGenerator.dialect.supports.dropTable.cascade) ? options.cascade
+    options.cascade = options.cascade !== null ? options.cascade
```

Comment:
```suggestion
    options.cascade = options.cascade != null ? options.cascade
```

Otherwise it doesn't cover undefined and the default value will not be applied unless cascade is set explicitly to null